src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueCalculator.kt

```diff
@@ -491,6 +494,26 @@ object EstimatedItemValueCalculator {
         return "§7$label: §e$have§7/§e$max ${price.formatCoinWithBrackets()}"
     }
 
+    private fun addPrestige(stack: ItemStack, list: MutableList<String>): Double {
+        val internalName = stack.getInternalNameOrNull() ?: return 0.0
+        if (!internalName.isKuudraArmor()) return 0.0
+        val tierIndex = internalName.getKuudraTier()?.takeIf { it > 1 } ?: return 0.0
+        val armorTier = kuudraTiers.getOrNull(tierIndex - 1) ?: return 0.0
+
+        val allTiersCost = (0 until tierIndex).mapNotNull { index ->
+            kuudraTiers.getOrNull(index)?.let { tierName ->
+                EstimatedItemValue.prestigeCosts[tierName]
+            }
+        }.sumByKey()
+
+        val (totalPrice, names) = getTotalAndNames(allTiersCost)
+
+        list.add("§7Tier: ${armorTier.lowercase().allLettersFirstUppercase()} ${totalPrice.formatCoinWithBrackets()}")
```

Comment:
please rename the title to something like "tier prestige" to make it clear for users that this is only the prestige cost and doesnt also include the starts cost. "but the tier price is starts plus prestige so this is wrong waahhh"